mmf/utils/visualize.py

```diff
@@ -0,0 +1,24 @@
+from typing import Any, List
+
+import torch
+import torchvision
+
+
+def visualize_images(images: List[Any], size=(224, 224), *args, **kwargs):
+    try:
+        import matplotlib.pyplot as plt
+    except ImportError:
+        print(
+            "Visualization tools require matplotlib. "
+            + "Installing using pip install matplotlib."
+        )
+        raise
+    transform = torchvision.transforms.Compose(
+        [torchvision.transforms.Resize(size=size), torchvision.transforms.ToTensor()]
+    )
```

Comment:
Do we always want to resize for visualization? Let's keep size `None` by default and then if someone wants to resize they can specify explicitly.

Reply:
We need to keep this as torchvision takes tensor as a default which needs to have same dimension. This can anyways be override, I can add an assert that if size is None, the can only be one image.